plugin/trino-iceberg/pom.xml

```diff
@@ -476,6 +476,7 @@
                         <requireUpperBoundDeps>
                             <excludes combine.children="append">
                                 <exclude>com.google.guava:guava</exclude>
+                                <exclude>com.fasterxml.jackson.core:jackson-databind</exclude>
```

Comment:
cc @electrum

Reply:
Airbase 131 is already on 2.14.1 and Trino and Trino master is on Airbase 132 (your branch is on 130),
so you can remove these excludes.

Reply:
ah great, thanks for letting me know. I've rebased the PR and removed those excludes.